bin/cyhy-kevsync

```diff
@@ -47,21 +47,29 @@ def parse_json(db, json_stream):
     data = json.load(json_stream)
     json_stream.close()
 
-    # Gather all the cveIDs from the JSON file
-    all_cve_ids = set()
+    # Gather cveID and knownRansomwareCampaignUse data from the JSON file
+    kevs = []
+    known_ransomware_count = 0
     for i in data["vulnerabilities"]:
         cve_id = i.get("cveID")
         if not cve_id:
             raise ValueError("JSON does not look like valid CISA KEV data.")
         else:
-            all_cve_ids.add(cve_id)
+            if i.get("knownRansomwareCampaignUse").lower() == "known":
```

Comment:
I'm pretty sure `get()` will return a `None` by default and it will fail when `lower()` is called.

```suggestion
            if i.get("knownRansomwareCampaignUse", "").lower() == "known":
```